cookbook_se/Makefile

```diff
@@ -45,12 +45,12 @@ fast_register_sandbox: fast_serialize_sandbox
 		--additional-distribution-dir s3://my-s3-bucket/fast/ --dest-dir /root/recipes ${CURDIR}/_pb_output/*
 
 .PHONY: fast_serialize_sandbox
-fast_serialize_sandbox:
+fast_serialize_sandbox: dev_docker_build
```

Comment:
should it still be called "sandbox" ?

Reply:
how about _in_container ?

Reply:
or _with_container

Reply:
just fast_serialize ?

Reply:
@EngHabu that's already a separate target we use for out of container serialize

Reply:
hmmm actually nvm... I thought, for some reason, that we already got rid of the different in container config files... hence why I was wondering why do we have multiple serialize_* targets to begin with... I then realized I was mistaken... so all good.. thanks for checking